.azure-pipelines/recover_testbed/recover_testbed.py

```diff
@@ -51,13 +51,12 @@ def recover_via_console(sonichost, conn_graph_facts, localhost, mgmt_ip, image_u
         elif device_type in ["nokia"]:
             posix_shell_onie(dut_console, mgmt_ip, image_url, is_nokia=True)
         else:
-            return
+            raise Exception("We don't support this type of testbed.")
```

Comment:
If testbed not support to recover by console, should we skip or fail?

Reply:
In my opinion, if we run this recover scipt, it means this testbed is unhealthy.  If the testbed is unhealthy, we don't need to execute next steps, so fail here.

Reply:
what about If the unhealthy reason is not unreachable? Would it pass the script or fail here?

Reply:
If ssh is reachable, we would not execute these code.

Reply:
Make sense to me, we used to let the console recover always return True, because it's in the pilot stage.
Not that it's stable enough, the mgmt-ip unavailable and not recoverable, we should raise exception and stop here,
no need to remove/add topo